frontend/package.json

```diff
@@ -80,13 +78,16 @@
     "url-loader": "^1.0.1",
     "webpack": "^4.20.2",
     "webpack-cli": "^3.1.1",
-    "webpack-dev-server": "^3.1.10",
+    "webpack-dev-server": "^3.1.14",
     "webpack-manifest-plugin": "^2.0.3",
     "webpack-merge": "^4.1.2"
   },
   "optionalDependencies": {
     "enzyme-adapter-react-16": "^1.1.1",
     "jest": "^22.4.3",
+    "jest-canvas-mock": "^1.1.0",
```

Comment:
It looks like you've removed canvas entirely? Is this unrelated to canvas?

Reply:
Yes and no. Canvas was needed by some lower level jsdom library that throws errors if tis not there. We don't actually use canvas anywhere. So this package just patches/mocks the canvas calls in jsdom so it doenst throw the error